server/handler/multisig.go

```diff
@@ -293,3 +293,92 @@ func (h *Handler) GetMultisigAccount(c echo.Context) error {
 		},
 	})
 }
+
+func (h *Handler) DeleteMultisigAccount(c echo.Context) error {
+	address := c.Param("address")
```

Comment:
We should not allow anyone to delete multisig account. Only the creator should be able to delete multsig.

Reply:
Authorization was added to the master branch. Once the code is merged, we will add middleware to protect the route.